ci/fireci/fireci/uploader.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import os
+import requests
+
+from fireci import prow_utils
+
+_logger = logging.getLogger('fireci.uploader')
+
+
+def post_report(
+    test_report,
+    metrics_service_url=os.getenv('METRICS_SERVICE_URL'),
+    access_token=prow_utils.gcloud_identity_token()):
+  """Post a report to the metrics service backend."""
+
+  endpoint = _construct_request_endpoint()
```

Comment:
I realise you have just moved this, but Consider using https://uplink.readthedocs.io/en/stable/ if you want to take a declarative approach to building the http client

Reply:
Just a consideration: Let's make sure we are confident in the future of uplink, i.e. requests is [`#4`](https://pypistats.org/top) most popular package on pypi while uplink is much less popular, so let's make sure it's not a personal project that is likely to get abandoned.

Reply:
Fair. Are there other packages that you are aware of that provide a retrofit like service?

Reply:
On second thought, this looks pretty good the way it is. No action required